scripts/test.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# This script is executed by Travis CI for every successful push (on any branch, PR or not).
+# It runs some basic tests on pages. If the build is also a PR, additional
+# checks are run through the check-pr script, and any message or error is sent
+# to tldr-bot to be commented on the PR.
+#
+# NOTE: must be run from the repository root directory to correctly work!
+# NOTE: `set -e` is applied conditionally only if needed.
+
+# Default test.
+function default_test {
```

Comment:
I'd suggest using the following function syntax instead:

```bash
default_test() {
    # ....
}
```

It's more portable :-)

Reply:
We're using the `function fname { ... }` syntax everywhere though. How is this more portable?

Reply:
Just looked it up, and I must be mistaken in this instance.

Ref https://ryanstutorials.net/bash-scripting-tutorial/bash-functions.php